src/components/icon/icon.js

```diff
@@ -1,19 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import Text from './components/text';
 
 import Element from '../element';
 
-const Icon = ({ size, color, className, align, text, ...props }) => {
```

Comment:
We can consider deprecating the `text` prop in v4.1 rather than outright removing it so people have time to migrate. We can then remove it in v5.